auth/src/main/res/layout/auth_method_picker_layout.xml

```diff
@@ -11,16 +11,28 @@
     <include layout="@layout/include_auth_method_picker_logo" />
 
     <LinearLayout
-        android:id="@+id/btn_holder"
-        style="@style/FirebaseUI.AuthMethodPicker.ButtonHolder.Portrait"
-        android:layout_height="wrap_content">
-
-        <Button
-            android:id="@+id/email_provider"
-            style="@style/FirebaseUI.Button.AccountChooser.EmailButton"
-            android:text="@string/sign_in_with_email"
-            android:visibility="gone"
-            tools:visibility="visible"/>
+      android:layout_height="wrap_content"
+      android:layout_width="wrap_content"
+      android:orientation="vertical">
+
+        <TextView
```

Comment:
It looks like this is just copy past from the landscape layout so we could use a `<merge />` tag for this.